common/tests/functional/Test-throughput.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+#Francis 
+if ! [ -x "$(type -P iperf3)" ]; then
+  echo "ERROR: script requires iperf3"
+  echo "For Debian and friends get it with 'apt-get install iperf3'"
+  echo "If you have it, perhaps you don't have permissions to run it, try 'sudo $(basename $0)'"
+  exit 1
+fi
+
+if [ "$#" -ne "2" ]; then
+  echo "ERROR: script needs four arguments, where:"
+  echo
+  echo "1. Number of times to repeat test (e.g. 10)"
+  echo "2. Host running 'iperf -s' (e.g. somehost)"
+  echo
+  echo "Example:"
+  echo "  $(basename $0) 10 somehost"
+  echo
+  echo "The above will run 'iperf -c' 10 times on the client and report totals and average."
+  exit 1
+else
+  runs=$1
+  host=$2
+fi
+
+log=qos_olsr.txt
+
+if [ -f "$log" ]; then
+  echo removing $log
+  rm $log
```

Comment:
Dangerous without double-quotes: for example $log may contain globbing characters such as *